sdks/python/apache_beam/coders/typecoders.py

```diff
@@ -92,6 +93,7 @@ def register_standard_coders(self, fallback_coder):
     self._register_coder_internal(bytes, coders.BytesCoder)
     self._register_coder_internal(bool, coders.BooleanCoder)
     self._register_coder_internal(str, coders.StrUtf8Coder)
+    self._register_coder_internal(windowed_value.PaneInfo, coders.PaneInfoCoder)
```

Comment:
This fix has a large blast radius. It changed registered standard coders which is default behavior globally. Shall we do this or just put a coder implementation in new ReShuffle?

Reply:
@kennknowles What do you think about this? We should use `PaneInfoCoder` but it could break the job update.

Reply:
Yes, we should update https://github.com/apache/beam/blob/master/CHANGES.md#breaking-changes-1 at least.